### DISCUSSION TITLE
请问会更新适配zotero 7的插件嘛？

### DISCUSSION THREAD

**Discussion by Chenmany** [018/zotcard#37] [Ideas]
开发大大好！请问会更新适配zotero 7的插件嘛？有这些模板真的好方便，可惜没适配zotero 7，之后会考虑升级嘛，谢谢~

**Comment by xiaoyezi-ysu** [comment]
同问，会考虑适配zotero7吗

**Comment by Paocaidacoco** [comment]
希望早日更新适配Zotero 7的版本，感恩了家人

**Comment by zzlb0224** [comment]
过来看看适配了吗？

**Comment by tina140119** [comment]
过来看看大大有没有更新适配7的版本，还是没有~ 继续等待 （因为舍不得zot card功能而一直酶没有更新到7）

**Comment by zzlb0224** [comment]
我是更新了7又退回去了周利斌周口师范学院 | Email: ***@***.*** | Tel: +08615690820302
